VideoLocker/res/layout/panel_find_course.xml

```diff
@@ -1,74 +1,77 @@
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_below="@+id/findcourse_tv"
-            android:layout_centerInParent="true"
-			android:padding="10dp"
-            android:background="@drawable/edx_brand_button"
-            android:text="@string/find_course_btn_text"
-            android:textAllCaps="true"
-            android:textColor="@color/white"
-            android:textSize="15sp" />
-        
-        <TextView
-            android:id="@+id/course_not_listed_tv"
-            style="@style/semibold_text"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_marginBottom="10dp"
-            android:layout_marginTop="17dp"
-            android:layout_below="@id/course_btn"
-            android:layout_centerHorizontal="true"
-            android:gravity="center"
-            android:linksClickable="true"
-            android:text="@string/course_not_listed_text"
-            android:textColor="@color/text_navigation"
-            android:textSize="14sp"
-            android:contentDescription="@string/course_not_listed_btn" />
-        
-    </RelativeLayout>
-    <!-- put a frame over this cell, so that corners look rounded -->
+            <TextView
+                android:id="@+id/findcourse_tv"
+                style="@style/semibold_text"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_centerHorizontal="true"
+                android:layout_marginBottom="18dp"
+                android:layout_marginTop="18dp"
+                android:gravity="center_horizontal"
+                android:text="@string/find_course_text"
+                android:textColor="#454951"
+                android:textSize="16sp" />
 
-    <ImageView
-        android:id="@+id/frame"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:layout_alignBottom="@id/panel_course_layout"
-        android:layout_alignLeft="@id/panel_course_layout"
-        android:layout_alignRight="@id/panel_course_layout"
-        android:layout_alignTop="@id/panel_course_layout"
-        android:background="@drawable/bg_frame_course" />
+            <Button
+                android:id="@+id/course_btn"
+                style="@style/regular_text"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_below="@+id/findcourse_tv"
+                android:layout_centerInParent="true"
+                android:background="@drawable/edx_brand_button"
+                android:padding="10dp"
+                android:text="@string/find_course_btn_text"
+                android:textAllCaps="true"
+                android:textColor="@color/white"
+                android:textSize="15sp" />
```

Comment:
We dont have this text size defined in `dimen.xml`, so we could either use one of the existing values or standardize this size.